crates/figma_import/src/toolkit_style.rs

```diff
@@ -15,14 +15,15 @@
 //! `toolkit_style` contains all of the style-related types that `toolkit_schema::View`
 //! uses.
 
+use crate::figma_schema;
 use dc_bundle::definition::layout::FlexWrap;
 use dc_bundle::legacy_definition::element::background::Background;
 use dc_bundle::legacy_definition::element::color::Color;
 use dc_bundle::legacy_definition::element::font::{
     FontFeature, FontStretch, FontStyle, FontWeight, TextDecoration,
 };
 use dc_bundle::legacy_definition::element::geometry::Size;
-use dc_bundle::legacy_definition::element::path::{LineHeight, Stroke};
+use dc_bundle::legacy_definition::element::path::{LineHeight, Path, Stroke};
```

Comment:
Do you know why `Path` is needed here?

Reply:
Looks like it wasn't, `cargo fix` removed it.

Maybe I should add a `cargo fix` presubmit...